wandb/sync/sync.py

```diff
@@ -111,6 +216,20 @@ def run(self):
                 result_q=result_q,
                 interface=publish_interface,
             )
+
+            if tb_root is not None:
+                if tb_event_files > 0 and sync_item.endswith(WANDB_SUFFIX):
```

Comment:
This condition confuses me. If the sync item is ends with `.wandb` (in which case it is a file) and therefore tb_root would be None and tb_event_files would be []. Guessing here, but is the check intended to be if any of the files under the `sync_item` (assuming `sync_item` is a dir) end in `WANDB_PREFIX`, then we abort the tf sync?

Reply:
Yeah, the thinking was if you call `wandb sync dir` and dir has both a "*.wandb" file and a "*.tfevent.*" file something is funky...  Although thinking about this now I should add a test for syncing an offline wandb run that did `sync_tensorboard=True`...